app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -1,3 +1,19 @@
+/* 
```

Comment:
It looks like we pull this one in via Blacklight so the header should be removed or replaced with one for blacklight.
https://github.com/projectblacklight/blacklight/blob/v6.20.0/app/assets/javascripts/blacklight/facet_load.js